bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/handler/SensiboAccountHandler.java

```diff
@@ -204,6 +205,8 @@ protected SensiboModel refreshModel() throws SensiboException {
     private <T> T sendRequest(final Request request, final AbstractRequest req, final Type responseType)
             throws SensiboException {
         try {
+            request.header(HttpHeader.ACCEPT_ENCODING, "gzip");
```

Comment:
Are you sure this is not already added by Jetty as default? Just to be sure this header is not present twice which could lead to requests being rejected if the server becomes more strict. See https://github.com/openhab/openhab-addons/issues/16982#issuecomment-2209531578

Reply:
You're right, it was always there! And yes, Jetty adds another header instead of overwriting the old one